playbooks/roles/wordpress/tasks/wordpress.yml

```diff
@@ -0,0 +1,126 @@
+---
+
+- name: Install dependencies
+  apt:
+    name: "{{ item }}"
+    update_cache: yes
+  with_items:
+    - unzip
+    - mysql-client
+    - build-essential
+    - python-dev
+    - libmysqlclient-dev
+    - python-mysqldb
+    - python-pip
+
+- name: Install the Python MySQLB module
+  pip:
+    name: mysqlclient
+
+- name: Ensure .ssh directory exists for www-data user
```

Comment:
@giovannicimolin, wouldn't it be better to do this all as a root/sudo user and then just setting the permissions to the `www-data` user at the end?